datasource/ethereum/src/block_ingestor.rs

```diff
@@ -60,10 +60,22 @@ where
                 static_self.do_poll().then(move |result| {
                     if let Err(e) = result {
                         // Some polls will fail due to transient issues
-                        warn!(
-                            static_self.logger,
-                            "Trying again after block polling failed: {:?}", e
-                        );
+                        match e {
+                            BlockIngestorError::BlockUnavailable(block_hash) => {
+                                trace!(
+                                    static_self.logger,
+                                    "Trying again after block polling failed: \
+                                    block data unavailable, block was likely uncled";
+                                    "block_hash" => format!("{:?}", block_hash)
```

Comment:
This should print with `{}` rather than `{:?}`.

Reply:
I used to have that, but H256's Display doesn't print the entire hash :(

Reply:
Ah, what I meant to say is that if we have `Fail` for BlockIngestError then maybe we should not be matching here and instead should just log `e`.